src/snowflake/cli/api/commands/flags.py

```diff
@@ -210,6 +238,51 @@ def _password_callback(value: str):
 )
 
 
+# consideration: it may be useful to put these options in a separate help panel, as they are somewhat generic among objects.
+# use these as your parameter names for IfExistsOption, IfNotExistsOption, and ReplaceOption to ensure that validation works
```

Comment:
Is there some way to add a "mutual exclusion" flag to the `OverrideableOption` so that `_create_mode_callback` doesn't have to depend on the exact parameter names? Or maybe a way to enforce the parameter names and fail if they weren't named properly?

Reply:
It is certainly possible to make this generic, though I do wonder what the future use cases of mutually exclusive options might look like. One way to format this might be to have a field in OverrideableOption that takes in a list of names that it is incompatible with and then dynamically form _create_mode_callback based on the passed names.

Reply:
Added a mutually exclusive field to OverrideableOption